README.md

```diff
@@ -49,7 +49,9 @@ var isEnabled = normalize(['string', 'boolean'], true);
 // Provide a function as first argument to do custom coercion
 var now = new Date();
 var enabledSince = normalize(function(value) {
-  return value.constructor === Date ? value : null;
+  if (value.constructor === Date) {
```

Comment:
I don't like ternaries and `undefined` is the return value if we don't return anything.  The only reason the docs had the ternary was due to requiring a `null` return.